docs/CHANGELOG.asciidoc

```diff
@@ -49,6 +50,20 @@ Review important information about the {kib} 8.x releases.
 * <<release-notes-8.0.0-alpha1>>
 
 --
+[[release-notes-8.10.2]]
+== {kib} 8.10.2
+
+The 8.10.2 release includes the following bug fixes.
+
+[float]
+[[fixes-v8.10.2]]
+=== Bug fixes
+
+Fleet::
+* Fixes force delete package, updated used by agents check ({kibana-pull}166623[#166623]).
+Management::
+* Fixes `isErrorResponse` when cluster details are provided ({kibana-pull}166544[#166544]).
```

Comment:
I think the title of this PR should be more aligned to the issue that's being fixed:

https://github.com/elastic/kibana/pull/166869

it's labeled "Cross cluster search causes errors when is_partial: true in the response"

So I would suggest something like 

"Fixes showing 'Received partial message' instead of results when there are some remote shard errors"

wdyt @lukasolson @davismcphee @thomasneirynck 
And we should apply this title + description to the fixing PR since it's an important one, so the relevant information should be visible at one place

Reply:
Thanks @kertal. I went with this suggestion for now. We can revert if wanted.

Reply:
@jrodewig I See you just applied my suggestion, I've added info that it happens when there are Cross cluster searches.

Reply:
@kertal Thanks again! 😄  I updated the title to include CCS with [`05ea420` (#166869)](https://github.com/elastic/kibana/pull/166869/commits/05ea420eb4454580b8cd06c1f7bd77dcda2cf09a)